### PR TITLE
Added graph-01.html

### LINKED ISSUE #1141: Create a Graph section (graph-01.html
One need to make this component using `HTML` and `Tailwind CSS`. I would suggest to use [Tailwind Playgrounds](https://play.tailwindcss.com/) to make things faster and quicker.

Here is a reference to the component. 
image

After building the component please raise a PR with a screenshot of the component and add the component in `seamless-ui/Graph/src/`. 
  
If you need to use any icon please use it from [Hero Icons](https://heroicons.com/)

Good luck.

The graph section UI will help user to visualize the data more clearly.
@Rajdip019

### PR DESCRIPTION
**Pull request by G0V1NDK**
Added graph-01.html to others section.
Additional chart.js was used while creating this component.

Closes issue: close #1141

Preview:
image

### CODE REVIEW

File: Others/graph-01.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
+<script src="https://unpkg.com/tailwindcss-jit-cdn"></script>
```
Comment (Rajdip019) [inline Others/graph-01.html:1]: Remove this line.

File: Others/graph-01.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+<section class="flex min-h-screen flex-col justify-center bg-gray-100 text-gray-600 antialiased">
+  <div class="mx-auto h-full max-w-3xl p-4 sm:px-6">
+    <div class="col-span-full flex flex-col rounded-sm border border-gray-200 bg-white shadow-lg xl:col-span-8">
+      <header class="flex items-center border-b border-gray-100 px-5 py-4">
+        <h2 class="font-semibold text-gray-800">Graph Analytics</h2>
+      </header>
+      <div class="flex justify-center align-middle">
+        <a href="#"><img src="https://i.ibb.co/2KBgLRP/Screenshot-2022-12-21-at-7-07-55-PM.png" alt="Screenshot-2022-12-21-at-7-07-55-PM" border="0" /></a>
```
Comment (Rajdip019) [inline Others/graph-01.html:10]: Don't wrap the image with a anchor tag. Remove the anchor tag.

File: Others/graph-01.html [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+              <div class="text-sm text-gray-500">Visit Duration</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<!-- imported chart from chart js -->
```
Comment (Rajdip019) [inline Others/graph-01.html:63]: Remove the whole script section.

**Comment by G0V1NDK** [comment]
@Rajdip019 done the required changes.

**Pull request merged**